python/tests/scripts/b1_equal_chain.metta

```diff
@@ -71,3 +71,11 @@
 !(assertEqualToResult
    (eq Green Blue)
   ((eq Green Blue)))
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+; `assert` checks if the result of evaluating an
+;  expression is True
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+!(assert (or True False))
+!(assert (< 5 8))
```

Comment:
Very minor, but the POSIX standard recommends to end text files with an empty new line.  That's why Github is showing this red forbidden symbol.